requirements.txt

```diff
@@ -1,6 +1,17 @@
+certifi==2017.11.5
+chardet==3.0.4
 click==6.7
 Flask==0.12.2
+idna==2.6
 itsdangerous==0.24
 Jinja2==2.10
 MarkupSafe==1.0
-Werkzeug==0.13
\ No newline at end of file
+numpy==1.13.3
+pandas==0.21.1
```

Comment:
Can we use the json format rather than rdb? Gut reaction is that pandas is overkill but I haven't looked at the code yet.

Reply:
> Can we use the json format rather than rdb?

I could be misreading, but I don't think water services supports json: https://waterservices.usgs.gov/rest/Site-Service.html#format.

Reply:
Shoot you are right. I still think pandas is overkill. I

Reply:
I hear the IV/DV services may be the only ones supporting JSON. Something about performance problems with it...

Agree that Pandas is a big dependency to pull in. This code is being used reliably in the retriever, but could be refactored to be more "Pythonic", and to return a dictionary instead of a tuple: https://code.chs.usgs.gov/dnaab/nwisweb-retriever/blob/cmdline/retriever/lib/rdb.py  Maybe a namedtuple would be an efficient data structure to unpack into, rather than a dictionary.

Reply:
@danielnaab -- I've only used namedtuples when I felt like I needed immutability. I'm not sure they offer much of an advantage over a dictionary in this case.